protocol-configuration/optimisticGovernance.ts

```diff
@@ -56,4 +36,23 @@ export const tribeCouncilPodConfig: PodConfig = {
   placeHolderMembers: placeHolderCouncilMembers
 };
 
-export const TRIBAL_COUNCIL_POD_ID = 25;
+export type PodConfig = {
```

Comment:
Worthwhile turning this into an interface rather than a type? I like the suggestion for objects

Reply:
Probably! I'm going to take a pass over all our types and interfaces in the next pr.